src/segment/myst_segment.rs

```diff
@@ -325,18 +339,37 @@ impl<R: Read + Seek> Loader<R, MystSegment> for MystSegment {
             tag_key_prefix: Rc::new(String::from(crate::utils::config::TAG_KEYS)),
             dedup: HashSet::new(),
             cluster: None,
+            duration: self.duration
         };
         Ok(Some(myst_segment))
     }
 }
 
 impl MystSegment {
+    pub fn new(shard_id: u32, epoch: u64) -> Self {
+        MystSegment::new_with_block_entries(shard_id, epoch, 50000 as usize)
+    }
+
+    pub fn new_with_block_entries(
+        shard_id: u32,
+        epoch: u64,
+        docstore_block_entries: usize,
+    ) -> Self {
+        MystSegment::new_with_block_entries_duration(shard_id, epoch, docstore_block_entries, 7200 as i32)
```

Comment:
let's have just one `new` method. A better option would be to have a MystSegmentBuilder, but we don't have too many optional parameters yet.

Reply:
Using plain new would mean coding default values in multiple places. I guess it can be done but,  a builder is a better option.